.github/workflows/merged-prs.yml

```diff
@@ -0,0 +1,36 @@
+name: "Add buildbot information to first PRs from new contributors"
+
+permissions:
+  contents: read
+
+on:
+  # It's safe to use pull_request_target here, because we aren't checking out
+  # code from the pull request branch.
+  # See https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+  pull_request_target:
+    types:
+      - closed
+
+jobs:
+  buildbot_comment:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: >-
+      (github.repository == 'llvm/llvm-project') &&
+      (github.event.pull_request.merged == true)
+    steps:
+      - name: Setup Automation Script
+        run: |
+          curl -O -L --fail https://raw.githubusercontent.com/"$GITHUB_REPOSITORY"/main/llvm/utils/git/github-automation.py
+          curl -O -L --fail https://raw.githubusercontent.com/"$GITHUB_REPOSITORY"/main/llvm/utils/git/requirements.txt
```

Comment:
Can you change this to use a sparse checkout like we do in https://github.com/llvm/llvm-project/blob/eca25bbdbb4f4681cd82de80e92b5f1315f0996a/.github/workflows/build-ci-container.yml#L39

Reply:
Done.

Would it be useful if I updated the other workflows that use curl also?

Reply:
(in new PRs ofc)

Reply:
Yes, that would be great.